pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -190,6 +190,9 @@ if [ -n "$VAULT_TLS_SERVER_NAME" ]; then
   ARGS+=(--connect-to ::"${VAULT_TLS_SERVER_NAME}":)
 fi
 
+# trim VAULT_BACKEND_PATH for last character '/' to avoid a redirect response from the server
+VAULT_BACKEND_PATH=$(echo "$VAULT_BACKEND_PATH"|sed 's|/$||')
```

Comment:
Why not use the bash builtin?

```sh
VAULT_BACKEND_PATH="${VAULT_BACKEND_PATH%/}"
```

Reply:
Because I wasn't aware of this one, previously looked at `foo=${foo///}` but it was removing all of them. Thanks!